typings/meta.d.ts

```diff
@@ -1,6 +1,7 @@
 export interface MetaSysProps {
   type: string,
   id: string,
+  appDefinition?: { sys: MetaLinkProps }
```

Comment:
I don't think we should be adding it to the sys type. I'd extend.

Reply:
@FLoppix